### ISSUE TITLE
Forum

### ISSUE THREAD

**Issue by happydasch** [backtrader2/backtrader#72]
Since it seems the community.backtrader.com page is dying i would ask you what you think of setting up a new forum where we at least can moderate the content.

@neilsmurphy  @vladisld  @ab-trader  and anyone else

**Comment by happydasch** [comment]
I would offer to set up something and link it here, also post regulary on community.backtrader.com to inform users of a new forum (regulary, since its not possible anymore to follow on anything on the current forum)

**Comment by neilsmurphy** [comment]
It's a reasonable suggestion. Is there any way to port the content?

**Comment by neilsmurphy** [comment]
btw, you can block users and it becomes usable pretty quickly.

**Comment by happydasch** [comment]
For thhe crawler solution, i am just afraid, that this may break some copyrights or other rights of the forum owner.

**Comment by neilsmurphy** [comment]
If Daniel would provide a dump, maybe he would open the forum up to admin access. I believe there are tools to block the spammers that are just not being implemented.

**Comment by happydasch** [comment]
> If Daniel would provide a dump, maybe he would open the forum up to admin access. I believe there are tools to block the spammers that are just not being implemented.

yes, when asking  for  a dump  its better to just ask for moderator rights :) would  be the easiest solution.

**Comment by neilsmurphy** [comment]
> > If Daniel would provide a dump, maybe he would open the forum up to admin access. I believe there are tools to block the spammers that are just not being implemented.
> 
> yes, when asking for a dump its better to just ask for moderator rights :) would be the easiest solution.

Cool, let us know how it goes.

**Comment by happydasch** [comment]
> The problem is how to contact Daniel. Anyone got his personal e-mail or other contact points ?

i am using a email which can be found online.

**Comment by happydasch** [comment]
I will keep you updated how it goes.

**Comment by Jake0303** [comment]
What about a discord? @happydasch good to see you, is btplotting in backtrader2?

**Comment by neilsmurphy** [comment]
> I will keep you updated how it goes.

@happydasch  Any update?

**Comment by happydasch** [comment]
Nothing happened since i sent the email. Maybe we can try to invite @mementum in here?

But anyway, i am still up for creating the forum. At the moment i do not have that much time and energy to set it up.

**Comment by mementum** [comment]
Hi guys,
I don't know which e-mail I was expected to receive, but I for sure didn't get it. In any case, I had a notification of a Javascript issue and I took the time to remove all (hopefully all) spam posts and the related accounts.

It would be no issue to make a couple of users moderators. Just let me know who.

Best regards

**Comment by happydasch** [comment]
@mementum i sent the text below to an email i found online i found online regarding the forum. Since github worked to notify you, its not current anymore. Thanks for your response. Since the most active user on the forum are @vladisld and @neilsmurphy, i would suggest to give these users moderator rights. If thats fine with both of you, too.

> I sent this email out:
> 
> Hello Daniel,
> 
> sorry to bother you. I am an user on your forum at community.backtrader.com. I follow your project for some time now and its sad, that the forum is slowly dying. It gets flooded with spam, it gets harder and harder to use it.
> 
> Since you seem to left the project for now I would like to ask if it would be possible to either provide some users moderator rights or provide a db dump so the awesome content of the forum is still available for users of backtrader. I am willing to either do some moderation on your forum, also it would be fine if we can move to a community forum and do not loose the collected knowledge.
> 
> What do you think?
> 
> Cheers,
> Daniel ...

**Comment by mementum** [comment]
It was in the spam folder which I don't regularly check for legit e-mails (I should do it more often I know)

@vladisld has been added. I would need the exact nickname @neilsmurphy uses on the forum to add him to the list too.

**Comment by mementum** [comment]
I guess the nickname is "run-out" ... it fits the pattern. Added

**Comment by vladisld** [comment]
Thanks Daniel,  very much appreciated !

**Comment by happydasch** [comment]
thanks a lot for a simple solution

**Comment by neilsmurphy** [comment]
@happydasch Thanks for working on tghis.

**Comment by vladisld** [comment]
And I wondered where all the spam authors have disappeared? The just haven't been able to open new accounts. :-)

**Comment by ab-trader** [comment]
That's why registration on the forum should be by invitation. :)
